### DISCUSSION TITLE
I have a problem fetching Received Documents.

### DISCUSSION THREAD

**Discussion by oraclecode318** [fattureincloud/api#483] [Q&A] [answered]
Hi, Everyone.
I tried to get the list of received documents by using api.

As you can see below, I have a received documents in my cloud service account.

![image](https://github.com/user-attachments/assets/19eb7a10-24e9-4acd-88c1-e88f1a4a2a72)

But when I try to fetch them by using api https://developers.fattureincloud.it/api-reference/#get-/c/-company_id-/received_documents

I can't see the any list of the received documents.
![image](https://github.com/user-attachments/assets/30206f38-c544-4a0d-8aab-4ddc58cbfa3a)

I used Received documents scope (both of read and write).

Please help me.

**Comment by barcigabri** [comment, marked as answer]
Hello,
As I can see from your first screenshot, you are trying to obtain your "to be registered" documents. However, the API call you are using in the second screenshot is for getting the expenses that you can find in FattureInCloud under the section Acquisitions->Expenses. Unfortunately, we don't have an API to obtain the documents, which is why you aren't getting any results.
I hope this solves your problem. Have a nice day.
Gabriele

**Reply by andreinl** [reply]
Hello,
can I ask you when if ever it will be possible to list "to be registered" documents? What is the reason for not providing this option?

Thank you in advance
Andrei
